src/main/java/ru/spb/devclub/utils/db/ResultSetUtils.java

```diff
@@ -1,4 +1,53 @@
 package ru.spb.devclub.utils.db;
 
+import java.sql.Date;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.text.DateFormat;
+
+@SuppressWarnings("WeakerAccess")
 public class ResultSetUtils {
+    public static <T> T getEnumValue(ResultSet rs, String columnName, T[] enumItems) throws SQLException {
+        String result = rs.getString(columnName);
+        if (!rs.wasNull()) {
+            for (T item : enumItems) {
+                if (item.toString().equals(result)) {
+                    return item;
+                }
+            }
+            throw new IllegalArgumentException("enum type has no constant " + result);
```

Comment:
```suggestion
            throw new IllegalArgumentException("Enum \"" + enumItems.getClass().getComponentType().getName() +
                    "\" doesn't contain a constant by the string value \"" + result + "\".");
```

The example from the test: 
`java.lang.IllegalArgumentException: Enum "ru.spb.devclub.utils.db.ResultSetUtilsTest$ExampleEnum" doesn't contain a constant by the string value "C".`